### ISSUE TITLE
customTemplates in model dont show

### ISSUE THREAD

**Issue by syphen** [greatCodeIdeas/md-date-range-picker#10]
I'm trying to add some custom templates and they don't seem to be showing when the calendar is instantiated, only the default templates are. Here's my code:
```
angular
.module('core')
.controller('baseCtrl', baseCtrl);

baseCtrl.$inject = ['$scope', '$mdDateRangePicker'];

function baseCtrl($scope, $datePicker) {
	var vm = $scope;

	vm.date = {
		showTemplate: true,
		customTemplates: [
			{
				name:"Last 3 Months",
				dateStart: new Date((new Date()).setMonth((new Date()).getMonth() - 3)),
				dateEnd : new Date()
			},{
				name:"Last 6 Months",
				dateStart: new Date((new Date()).setMonth((new Date()).getMonth() - 6)),
				dateEnd : new Date()
			}
		]
	}
	vm.showDate = function($event, showTemplate){
		$datePicker.show({targetEvent:$event, model:vm.date} ).then(function(result){
			if(result) vm.date = result;
		})
	}
}
```

Also, is there a way for this to work and is there a way for me to disable the default templates?

**Comment by roelbarreto** [comment]
Hi @syphen Can you check the demo site https://ipiz.herokuapp.com/md-date-range-picker-demo/index.html where custom templates are added
We will add feature to disable the predefined templates

**Comment by roelbarreto** [comment]
Released version 0.4.1 to resolve this issue
Tanks to @yoni12ab
